gen-ai/orchestrator-server/src/main/python/server/src/gen_ai_orchestrator/models/compressors/flashrank_rerank/flashrank_rerank_params.py

```diff
@@ -0,0 +1,43 @@
+#   Copyright (C) 2023-2024 Credit Mutuel Arkea
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+"""Model for creating FlashrankRerankCompressorSetting."""
+
+from typing import Literal, Optional
+
+from pydantic import Field
+from langchain.retrievers.document_compressors import FlashrankRerank
+from gen_ai_orchestrator.models.compressors.compressor_provider import (
+    CompressorProvider,
+)
+from gen_ai_orchestrator.models.compressors.document_compressor_params import (
+    BaseCompressorParams,
+)
+
+
+class FlashrankRerankCompressorParams(BaseCompressorParams):
+    provider: Literal[CompressorProvider.FLASHRANK_RERANK] = Field(
+        description='The Flashrank Rerank Model Provider.',
+        examples=[CompressorProvider.FLASHRANK_RERANK],
+        default=CompressorProvider.FLASHRANK_RERANK.value
+    )
+    model: Optional[str] = Field(description='The model id', examples=[],
```

Comment:
Add to the description a link to flashrank rerank list of available models : https://github.com/PrithivirajDamodaran/FlashRank?tab=readme-ov-file#features